test/createReducer.spec.js

```diff
@@ -0,0 +1,33 @@
+import { createReducer, fromAction, getState } from '../src'
+
+describe('createReducer', () => {
+  const dependencies = ['foo', 'bar', 'baz'].map(fromAction).concat(getState)
+  const state = 'test'
+  const [foo, bar, baz] = ['fooVal', 'barVal', 'bazVal']
+
+  it(`Accepts a list of branching functions and converging function and returns a new function. When invoked, this new function is applied to some arguments, each branching function is applied to those same arguments. The results of each branching function are passed as arguments to the converging function to produce the return value.`, () => {
```

Comment:
Is there any way to break the description of this test down or make it simpler to read?

Reply:
I'm open to suggestions :smile:

Reply:
Yes... that's quite hard. I think you can completely skip the first sentence, and try to make the rest smaller. This is the best I could think of:

```
it('creates a function that will return computeFn called with the result of executing each of the dependencies with the original arguments')
```

Still not 100% happy with this.

Reply:
Ok, what about this?
```suggestion
  it(`receives 2 parameters: an Array of dependency functions and a compute function. It returns a function that invokes the compute function using the results of the dependencies as the arguments`, () => {
```